src/woo_search/search_index/client.py

```diff
@@ -117,6 +117,17 @@ def clean_query(query: str) -> str:
     return query.replace("&&", r"\&&").replace("||", r"\||")
 
 
+def bucket_filter(
```

Comment:
This will not work how I want it to with more then 2 bucket_filters. I will have to see if I can group all the AND filters together.